### ISSUE TITLE
assert d in name_to_node, "%s is not in graph" % d

### ISSUE THREAD

**Issue by rokij6698** [tensorflow/tensorflow#21878]
When I convert the keras model to pb file, it reports an error that :

assert d in name_to_node, "%s is not in graph" % d

Does anyone know any solution of this issue? Thank you.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
Mobile device

**Comment by tensorflowbutler** [comment]
It has been 20 days with no activity and the `awaiting response` label was assigned. Is this still an issue?
